packages/components/rate/__tests__/rate.vitest.tsx

```diff
@@ -83,10 +83,10 @@ describe('Rate.vue', () => {
       <Rate v-model={value.value} colors={['#99A9BF', '#F7BA2A', '#FF9900']} />
     ))
 
-    const thirdStar = (
-      wrapper.findAll('.el-rate__item')[2].element as HTMLElement
-    ).querySelector<HTMLElement>('.el-rate__icon')!
-    expect(thirdStar.style.color).toMatchInlineSnapshot('"rgb(255, 153, 0)"')
+    const rateEl = wrapper.find('.el-rate').element as HTMLElement
+    expect(rateEl.style.getPropertyValue('--el-rate-fill-color')).toBe(
```

Comment:
How about using `toMatchInlineSnapshot`

Reply:
Maybe it's simpler for using `toBe` directly.

> It’s typically better to make more specific assertions than to use snapshots. These kinds of tests include implementation details so they break easily, and teams can get desensitized to snapshot breakages. Selectively [mocking some child components](https://reactjs.org/docs/testing-recipes.html#mocking-modules) can help reduce the size of snapshots and keep them readable for the code review.

> https://reactjs.org/docs/testing-recipes.html#snapshot-testing